src/main/java/frc/robot/subsystems/turretSubsystem/TurretCommandFactory.java

```diff
@@ -27,16 +32,31 @@ public class TurretCommandFactory {
   final RobotStateManager RSM;
   final VisionSubsystem vision;
   final Supplier<Rotation2d> rotationSupplier;
+  final Supplier<Translation2d> translationSupplier;
+  static final InterpolatingDoubleTreeMap pitchMap;
+
+  DebugEntry<Double> limelightDistance;
+
+  static {
+    pitchMap = new InterpolatingDoubleTreeMap();
+    for (TurretDataPoint TDP : TurretConstants.TURRET_DATA) {
+      pitchMap.put(TDP.limelightMeters(), TDP.turretAngleRadians());
+    }
+  }
```

Comment:
Does pitchMap need to be static? If it doesn't this can be done in the constructor